.devcontainer/Dockerfile

```diff
@@ -42,4 +50,26 @@ RUN conan install . \
 -pr:a .conan/gcc/profile
 
 RUN useradd -ms /bin/bash sourcetrail
-USER sourcetrail
\ No newline at end of file
+USER sourcetrail
+
+WORKDIR /home/sourcetrail
+COPY --chown=sourcetrail:sourcetrail . /home/sourcetrail
+
+RUN if [ "$COMPILER" = "gcc" ] || [ "$COMPILER" = "clang" ]; then \
+      conan install . -s build_type=Release -of .conan/${COMPILER}/ -b missing -pr:a .conan/${COMPILER}/profile; \
+    else \
+      echo "Unknown compiler"; \
+      exit 1; \
+    fi
+  
+RUN if [ "$COMPILER" = "gcc" ]; then \
+      cmake --preset=gnu_release; \
+    else \
+      cmake --preset=clang_release; \
+    fi
+
+WORKDIR /home/sourcetrail/build/Release/$(if [ "$COMPILER" = "gcc" ]; then echo "GNU"; else echo "CLang"; fi)
+
+RUN ninja
+
+CMD ["./Sourcetrail"]
```

Comment:
No need. It's a developement container not a production one. It should work with vs-code.